### PR TITLE
User assignee instructions

### PR DESCRIPTION
**Pull request by amberben**
The security signal side panel now supports assigning users to signals for CWS and Cloud SIEM. Added directions to this.

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/security_signal_management/_index.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ To modify the signal state, enable the “Security Signals Write” permission f
 
 ## View and respond to Security Signals
 
-If Datadog detects a threat based on any rules, it creates a security signal. You can view all security signals in the Signal Explorer, making it easy to search, filter, and correlate them without needing to learn a dedicated query language. You can either monitor signals from the Datadog Cloud Security Platform, or you can configure [Notification Rules][3] to send signals to third-party tools. The status of a signal can be one of the following:
+If Datadog detects a threat based on any rules, it creates a security signal. You can view all security signals in the Signal Explorer, making it easy to search, filter, and correlate them without needing to learn a dedicated query language. You can either monitor signals from the Datadog Cloud Security Platform, or you can configure [Notification Rules][3] to send signals to third-party tools. Signals can be assigned to yourself or to another user in the Datadog platform.
```
Comment (maycmlee) [inline content/en/security_platform/security_signal_management/_index.md:19]:
```suggestion
A Security Signal is created when Datadog detects a threat based on a security rule. View, search, filter, and correlate security signals in the Signal Explorer without needing to learn a dedicated query language. You can either monitor signals from the Datadog Cloud Security Platform, or configure [Notification Rules][3] to send signals to third-party tools. Signals can be assigned to yourself or another user in the Datadog platform.
```

File: content/en/security_platform/security_signal_management/_index.md [hunk 2]
```diff
@@ -37,7 +39,8 @@ To change signal state, follow the instructions below:
 
 1. In Datadog, select Security from the main navigation menu on the left, and select Security Signals from the menu.
 2. Select a Log Detection or Workload Security Signal from the table.
-3. Navigate to the top-right corner of the Signal Side Panel and select your desired status from the dropdown menu. The default status is **Open**.
+3. To assign a signal to yourself or another Datadog user, navigate to the user profile icon with the plus sign in the top-right corner of the Signal Side Panel.
```
Comment (maycmlee) [inline content/en/security_platform/security_signal_management/_index.md:42]:
```suggestion
3. To assign a signal to yourself or another Datadog user, navigate to the user profile icon with the plus sign in the top right corner of the Signal side panel.
```

File: content/en/security_platform/security_signal_management/_index.md [hunk 3]
```diff
@@ -37,7 +39,8 @@ To change signal state, follow the instructions below:
...
+4. Navigate to the top-right corner of the Signal Side Panel and select your desired status from the dropdown menu. The default status is **Open**.
```
Comment (maycmlee) [inline content/en/security_platform/security_signal_management/_index.md:43]:
```suggestion
4. Navigate to the top right corner of the Signal side panel, and select your desired status from the dropdown menu. The default status is **Open**.
```

**Review by maycmlee — COMMENTED** [review]
Left some feedback!

**Comment by amberben** [comment]
It looks good! thanks @maycmlee !

**Comment by amberben** [comment]
Should I "close with comment"? @maycmlee

**Comment by maycmlee** [comment]
Thanks @amberben for adding this! I'll approve and merge this so that it's in the docs. If you close the PR, it won't get merged.

**Review by maycmlee — APPROVED** [review]

**Pull request merged**